integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -457,7 +466,8 @@ case class CarbonLoadDataCommand(
   private def loadDataWithPartition(sparkSession: SparkSession,
       carbonLoadModel: CarbonLoadModel,
       hadoopConf: Configuration,
-      dataFrame: Option[DataFrame]) = {
+      dataFrame: Option[DataFrame],
+      operationContext: OperationContext) = {
```

Comment:
either delete line 460 to 464, or add the description for them

Reply:
ok, removed